packages/jupyter-ai/jupyter_ai/__init__.py

```diff
@@ -33,3 +34,7 @@ def _jupyter_server_extension_points():
 
 def load_ipython_extension(ipython):
     ipython.register_magics(AiMagics)
+    ipython.set_custom_exc((BaseException,), store_exception)
+
+def unload_ipython_extension(ipython):
+    ipython.set_custom_exc((BaseException,), ipython.CustomTB)
```

Comment:
Could you provide some authoritative reference that demonstrates `ipython.CustomTB` is the default exception handler? It's OK if it's just code; I can't find any mention of it in the docs.

Reply:
@ellisonbg provided me with guidance to use `ipython.CustomTB` as the fallback handler, although I can also remove the `unload_ipython_extension` function if it's not needed here.